tests/unit/data_processing/test_data_processor.py

```diff
@@ -90,6 +97,9 @@ def test___init__(self, update_transformer_mock, mock_rdt):
         assert data_processor._hyper_transformer == mock_rdt.HyperTransformer.return_value
         update_transformer_mock.assert_called_with(True, False)
 
+        mock_default_transformers.assert_called_once()
```

Comment:
you should mock the return value and make sure the id default is properly added and that the text one is deleted

Reply:
Yes done in [d84def8](https://github.com/sdv-dev/SDV/pull/1506/commits/d84def8dca6ca34f899c558cc4bc04d17fb01641)